test/integration/targets/delegate_to/test_delegate_to.yml

```diff
@@ -1,9 +1,11 @@
 - hosts: testhost3
```

Comment:
Mainly just having the test figure out that the IPs are defined and auto-skip the test if they aren't (IIRC you said you had to preconfigure the IPs on your box to get the test to run locally).